src/app/zap-templates/zcl/data-model/chip/dishwasher-control-cluster.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Appliances"/>
+
+  <bitmap name="DishwasherControlFeature" type="BITMAP32">
+    <cluster code="0x0058" />
+    <field name="STEAMWASH" mask="0x01" />
+    <field name="HEATEDDRY" mask="0x02" />
+    <field name="EXTDRY" mask="0x04" />
+  </bitmap>
+
+  <cluster>
+    <domain>Appliances</domain>
+    <name>Dishwasher Control</name>
+    <code>0x0058</code>
+    <define>DISHWASHER_CONTROL_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides a way to access options associated with the operation of a Dishwasher device type.</description>
+
+    <globalAttribute side="server" code="0xFFFD" value="1" />
+
+    <attribute side="server" code="0x0000" define="AVAILABLE_OPTIONS_FOR_CURRENT_MODE" type="BITMAP32" entryType="INT32U" min="0x00" max="0x07"      writable="false" isNullable="false" optional="false">AvailableOptionsForCurrentMode</attribute>
```

Comment:
```suggestion
    <attribute side="server" code="0x0000" define="AVAILABLE_OPTIONS_FOR_CURRENT_MODE" type="BITMAP32" min="0x00" max="0x07"      writable="false" isNullable="false" optional="false">AvailableOptionsForCurrentMode</attribute>
```
Should there be an actual bitmap type defined for this in the spec (and here)?  But for now, matching the spec is the right thing.